pkg/database/realm.go

```diff
@@ -52,6 +52,7 @@ const (
 	TestTypeConfirmed
 	TestTypeLikely
 	TestTypeNegative
+	TestTypeUserReport
```

Comment:
I _think_ there's another place where we enumerate these constants (templates maybe?). I can't remember right now, but I think you need to add something somewhere else too.

Reply:
Edit: nevermind - it's vercode.go and you did it already.

Reply:
yep - all good.